doc/vmware_workstation_15_5_0_setup.rst

```diff
@@ -0,0 +1,63 @@
+
+For WMware worktation 15.5.0
+============================
+
```

Comment:
Why do we need this document, while the "WMware worktation 15.5.0" is not the latest official release ?

Reply:
Latest version only work with the latest version of Windows. I can't update to latest version of Windows because of my house line issue, and I no longer back to office since last year due to covid.